packages/experiments/src/components/Stack/Stack.types.ts

```diff
@@ -44,6 +44,13 @@ export interface IStackProps
    */
   horizontal?: boolean;
 
+  /**
+   * Defines whether to render Stack child elements in the opposite direction (bottom-to-top if it's a Vertical Stack and
```

Comment:
Very nit-picky nit here: should the words "Vertical" and "Horizontal" be capitalized if they're no longer part of the Stack class names (we no longer have VerticalStack and HorizontalStack)?